src/client/components/Search/Search.styles.css

```diff
@@ -0,0 +1,24 @@
+.search {
+  background-color: #f2f9fb;
+  border: none;
+  border-radius: 10px;
+  width: 343px;
+  height: 40px;
+  left: 16px;
+  top: 80px;
+}
+
+input {
+  padding: 10px;
+}
+
+.userLinkInSearch {
+  text-decoration: none;
+}
+
+.cancelInputSearch {
+  background-color: white;
+  color: #4c9999;
+  border: none;
+  margin-left: 5px;
+}
```

Comment:
Unused CSS styles.

Reply:
.userLinkInSearch uses for making user name without default style. (when you see a name under the search field. But I remove it in case someone else will decorate it.

.cancelInputSearch uses for not seeing a button style.  as we have in a prototype in Figma